### PR TITLE
Optimized the indentation of codes and fix two typos

### PR DESCRIPTION
**Pull request by miaoyq**
Signed-off-by: Yanqiang Miao <miao.yanqiang@zte.com.cn>

### CODE REVIEW

**Review by thaJeztah — APPROVED** [review]
LGTM, thanks!

/cc @vdemeester @dnephin

**Comment by duglin** [comment]
where are the two typos?

**Comment by dnephin** [comment]
My guess is that it was the uppercase letter in the error message.

**Review by dnephin — APPROVED** [review]
LGTM

File: daemon/daemon_unix.go [hunk 1]
```diff
@@ -160,29 +160,30 @@ func parseSecurityOpt(container *container.Container, config *containertypes.Hos
...
 
-			if len(con) != 2 {
-				return fmt.Errorf("Invalid --security-opt 1: %q", opt)
-			}
+		var con []string
+		if strings.Contains(opt, "=") {
+			con = strings.SplitN(opt, "=", 2)
+		} else if strings.Contains(opt, ":") {
+			con = strings.SplitN(opt, ":", 2)
+			logrus.Warn("Security options with `:` as a separator are deprecated and will be completely unsupported in 1.13, use `=` instead.")
```
Comment (duglin) [inline daemon/daemon_unix.go]: Should this be "security" instead of "Security" then?

Comment (dnephin) [inline daemon/daemon_unix.go]: Nope, this is a warning printed directly to stderr, not an error message (that could be append to some other string).

Comment (duglin) [inline daemon/daemon_unix.go]: never really understood/liked that pattern, but ok :-)

**Comment by duglin** [comment]
LGTM

**Pull request merged**